examples/example-shader/src/lib.rs

```diff
@@ -168,7 +190,15 @@ pub fn main_fs(input: Input<Vec4>, mut output: Output<Vec4>) {
     // evaluate Preetham sky model
     let color = sky(dir, sun_pos);
 
-    output.store(color.extend(0.0))
+    color.extend(1.0)
+}
+
+#[allow(unused_attributes)]
+#[spirv(entry = "fragment")]
+pub fn main_fs(input: Input<Vec4>, mut output: Output<Vec4>) {
+    let v = input.load();
+    let color = fs(Vec2::new(v.0, v.1));
```

Comment:
This makes me think that we should talk about the unwrapping of Inputs/Outputs more - I've flipflopped back and forth on this, there's arguments both ways. Also, we should be able to construct Intputs/Outputs on the CPU, right? (maybe need to make something `pub`, `pub fn construct_from_cpu_var(&mut T) -> Output<T>` or something)

Reply:
Yes the current `Input` and `Output` was not ideal for this, but do think it is good to have this use case at least and we can think about the design separately